### PR TITLE
PSR-1661 Fixed pagination

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

File: app/views/ListView.scala.html [hunk 1]
```diff
@@ -55,8 +55,12 @@ <h1 class="govuk-heading-l">@renderMessage(viewmodel.heading)</h1>
                 <h1 class="govuk-heading-l">@renderMessage(viewmodel.optViewOnlyDetails.get.heading)</h1>
             }
 
+            @if(viewmodel.page.sections.size == 1){
```
Comment (kberec) [inline app/views/ListView.scala.html:58]: Probs this is not needed, please check

File: .gitignore [hunk 2]
```diff
@@ -35,3 +35,8 @@ bin
 *.iws
 /.bsp/sbt.json
 /.java-version
+.vscode/
```
Comment (kberec) [inline .gitignore:38]: Are these needed?

Comment (westwater) [reply .gitignore:38]: yeah trying out vscode

File: app/views/ListView.scala.html (hunk 1 above)
Comment (westwater) [reply app/views/ListView.scala.html:58]: woops leftover from initial impl

**Review by kberec — APPROVED** [review]

**Pull request merged**